src/main.rs

```diff
@@ -10,7 +10,10 @@ use libc::c_char;
 
 macro_rules! clean_panic
 {
-    ($str:literal) => { panic!("{}", $str); }
+  ($str:literal) => {
+    println!("Error: {}", $str);
+    std::process::exit(1)
```

Comment:
Panicing was done on purpose please restore those semantics. Returning with 1 is unacceptable as that amounts to successful termination in bash procedures.